test/src/common/harnessHost.h

```diff
@@ -463,6 +465,7 @@ typedef struct HrnHostConfigUpdateParam
 {
     VAR_PARAM_HEADER;
     const Variant *archiveAsync;                                    // Update async archiving?
+    const Variant *spoolPath;                                       // Spool path
```

Comment:
Do we need a comment that repeats the field name? Maybe we can remove this, or make it more detailed